src/libraries/Native/Unix/System.Globalization.Native/configure.cmake

```diff
@@ -1,32 +1,37 @@
-include(CheckCSourceCompiles)
-include(CheckSymbolExists)
-
 if(CLR_CMAKE_TARGET_ANDROID)
-    string(REPLACE ";" ":" ANDROID_RPATHS "${CMAKE_C_IMPLICIT_LINK_DIRECTORIES}:${CMAKE_SYSTEM_LIBRARY_PATH}")
-    set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -rpath ${ANDROID_RPATHS}")
-endif()
+    set(HAVE_SET_MAX_VARIABLE 1)
+    set(HAVE_UDAT_STANDALONE_SHORTER_WEEKDAYS 1)
+else()
+    include(CheckCSourceCompiles)
+    include(CheckSymbolExists)
+
+    if(CLR_CMAKE_TARGET_ANDROID)
```

Comment:
Will this ever be executed? This is inside an else for `if(CLR_CMAKE_TARGET_ANDROID)`.

Reply:
Hm yeah this is from @am11's work which uses a downloaded ICU library. I think my code to load the system ICU should work for his case too so I can probably get rid of this here.

Reply:
I removed this check for now, we can add it back later if it's still needed :)